app/src/main/java/com/fruitscale/ananas/MainActivity.kt

```diff
@@ -158,4 +169,15 @@ class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelecte
 
         return true
     }
+
+    override fun onServiceDisconnected(name: ComponentName) {
```

Comment:
This is needed to bind the activity to the session service, for retrieving Matrix sessions